critiquebrainz/db/review.py

```diff
@@ -40,6 +40,21 @@ def to_dict(review, confidential=False):
     return review
 
 
+def get_top_languages():
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+        SELECT  language, 
```

Comment:
Make sure that the you make an indent here. See PEP8 [warning](https://ci.metabrainz.org/job/cb-additional/303/warnings44Result/new/package.1207085015/author.710252144/source.1688975576924129965/#36) from the tests. Also, make sure that the SQL follows guidelines (All the SELECT, FROM statements must be properly aligned): https://github.com/metabrainz/guidelines/blob/master/Python.md#sql